src/shared/translation/translationEnglish.json

```
@@ -1575,5 +1575,96 @@
+    {
+      "header": "Learning Elements",
+      "body": [
+        "Production of Learning Elements in the field of Software Engineering with with a multi stage review process",
+        "Conception of a literature based category system for the typification of the learning elements"
+      ]
+    },
+    {
+      "header": "Learning Analytics",
+      "body": [
+        "Collection and analysis of learning analytics with persistence in a learning record store (data lake)",
+        "Recording of student behavior through Moodle and frontend in the form of statements according to the xAPI standard",
+        "Creation and implementation of designs for student dashboards"
+      ]
+    },
+    {
+      "header": "Connection to Moodle",
+      "body": [
+        "Development of multiple Moodle Plugins (xAPI, Moodle Theme and H5P-Report Feature)",
+        "Communication with Moodle using the provided REST-API interface. ",
+        "LTI based communication between frontend and Moodle"
+      ]
+    },
+    {
+      "header": "Frontend and Backend Development",
+      "body": [
+        "React-Frontend-development (modular development through Components and Pages)",
+        "Jest-Frontend Testing and Pytest-Backend Testing",
+        "Development of a CI/CD pipeline for frontend and backend in GitHub using SonarLint",
+        "Python-Backend-development",
+        "Communication between frontend and backend using API endpoints with persistence in a Postgres database",
+        "Branch-focused development with review process on GitHub",
+        "Code based inline source documentation in the form of comments in Frontend and Backend"
+      ]
+    },
+    {
+      "header": "Conception of a learning system",
+      "body": [
+        "Theoretical preliminary conceptualization of an adaptive learning system",
+        "Evaluation algorithms for various questionnaires (Index of Learning Styles - ILS and Lernstrategien im Studium, shortened - LIST-K)",
+        "Design, implementation and application of AI algorithms (e.g. ant colony algorithm and genetic algorithm) to generate individualized learning paths in a multidimensional hypercube",
+        "Implementation and application of literature-based algorithms based on the learning style (e.g. according to Graf et al. 2010)",
+        "Further development of reliable, validated and shortened questionnaire versions of the ILS and the LIST-K using e.g. factor analysis, exhaustive feature selection and Bayesian networks",
+        "Collaboration between frontend, backend and Moodle for the automatic generation of learning paths for students",
+        "Modeling the individual level of knowledge using a rating system",
+        "Modeling student characteristics such as learning style and learning strategy",
+        "Literature research about Adaptiven Lernsystemen, Learning Analytics, Knowledge Tracing, Rating Systems, Large Language Models und Natural Language Processing",
```

Eigennamen groß, rest klein
```suggestion
        "Literature research on adaptive learning system, learning analytics, Knowledge Tracing, rating systems, large language models und natural language processing",
```